frontend/src/modules/auth/pages/LoginPage.js

```diff
@@ -0,0 +1,15 @@
+import ContentBox from '../../../shared/design-system/atoms/ContentBox';
+import { RouterLink } from '../../../shared/navigation';
+import { Text } from '@chakra-ui/react';
+
+const LoginPage = () => {
+  return (
+    <ContentBox>
+      <Text>Tady se uživatel přihlásí</Text>
+      <br></br>
```

Comment:
Nepoužívej br, dej tam styl + není to self-closing (ale v tomhle případě je to asi jedno, když se to stejně bude měnit)

Reply:
Jasnačka <br></br> -> <br /> já to tam hodil jenom kvůli tomu, že je to dočasný